numpy/core/tests/test_datetime.py

```diff
@@ -1118,11 +1118,15 @@ def test_datetime_compare_nat(self):
                     assert_warns(FutureWarning, op, td_other, td_nat)
 
             assert_warns(FutureWarning, np.not_equal, dt_nat, dt_nat)
```

Comment:
Hmm, looking at this, it looks like we are only testing scalars, although calling the ufunc turns them into 0-D arrays, so strictly speaking I suppose we don't test scalars. Would probably need to use the `==` and `!=` operators. I note that those operators also appear to call the ufunc, so probably not worth a lot of trouble as the behavior will change in the future.